### ISSUE TITLE
Import log says "None" in album-grouping mode

### ISSUE THREAD

**Issue by smcvb** [sampsyo/beets#825]
Found this issue while importing a large music set from which I only want the tagged data in my database, so not alter the files (move/rename) in any fashion. I didn't fancy taking every auto-tag issue into account, hence I imported the files in quite mode and logged the music files which it couldn't tag so I could do those by hand later. However, since the log only contained "Skipped none" and I did not move or alter any of my files, I had no way of telling which files weren't logged.

In short, when you are importing music in quite mode and want to log which files are skipped, the log only contains lines like "Skipped none".
